app/controllers/collections_controller.rb

```diff
@@ -9,7 +9,6 @@ class CollectionsController < ApplicationController
 
   def show
     authorize! @collection
-    @status_presenter = StatusPresenter.new(status: @status)
```

Comment:
StatusPresenter is being changed to VersionStatus model. VersionStatus is being returned directly be Sdr::Repository.status().

This is many of the changes below.